packages/panzer/adapters-stk/test/stk_interface_test/CMakeLists.txt

```diff
@@ -146,6 +146,12 @@ IF (PANZER_HAVE_EPETRA_STACK)
     SOURCES   tScatterSpeed_Epetra.cpp
   )
 ENDIF()
+# Disabled until further guidance
```

Comment:
@rppawlo This is the example that I disabled (it doesn't compile and I didn't pursue it yet). Note: when I tried to run its epetra counterpart I got a segfault. I'm not sure how these examples are used.